docs/src/modules/components/ComponentLinkHeader.js

```diff
@@ -1,199 +1 @@
-            clickable
-            role={undefined}
-            component="a"
-            size="small"
-            variant="outlined"
-            rel="nofollow"
-            href={headers.materialDesign}
-            icon={<MaterialDesignIcon />}
-            data-ga-event-category="ComponentLinkHeader"
-            data-ga-event-action="click"
-            data-ga-event-label="Material Design"
-            data-ga-event-split="0.1"
-            label="Material Design"
-          />
-        </li>
-      ) : null}
-      {design !== false ? (
-        <React.Fragment>
-          <li>
-            <Chip
-              clickable
-              role={undefined}
-              component="a"
-              size="small"
-              variant="outlined"
-              rel="nofollow"
-              href="https://mui.com/store/items/figma-react/?utm_source=docs&utm_medium=referral&utm_campaign=component-link-header"
-              icon={<FigmaIcon />}
-              data-ga-event-category="ComponentLinkHeader"
-              data-ga-event-action="click"
-              data-ga-event-label="Figma"
-              data-ga-event-split="0.1"
-              label="Figma"
-            />
-          </li>
-          {packageName !== '@mui/joy' ? (
-            <li>
-              <Chip
-                clickable
-                role={undefined}
-                component="a"
-                size="small"
-                variant="outlined"
-                rel="nofollow"
-                href="https://mui.com/store/items/adobe-xd-react/?utm_source=docs&utm_medium=referral&utm_campaign=component-link-header"
-                icon={<AdobeXDIcon />}
-                data-ga-event-category="ComponentLinkHeader"
-                data-ga-event-action="click"
-                data-ga-event-label="Adobe XD"
-                data-ga-event-split="0.1"
-                label="Adobe"
-              />
-            </li>
-          ) : null}
-          {packageName !== '@mui/joy' ? (
-            <li>
-              <Chip
-                clickable
-                role={undefined}
-                component="a"
-                size="small"
-                variant="outlined"
-                rel="nofollow"
-                href="https://mui.com/store/items/sketch-react/?utm_source=docs&utm_medium=referral&utm_campaign=component-link-header"
-                icon={<SketchIcon />}
-                data-ga-event-category="ComponentLinkHeader"
-                data-ga-event-action="click"
-                data-ga-event-label="Sketch"
-                data-ga-event-split="0.1"
-                label="Sketch"
-              />
-            </li>
-          ) : null}
-        </React.Fragment>
-      ) : null}
-    </Root>
-  );
-}
-
-ComponentLinkHeader.propTypes = {
-  design: PropTypes.bool,
-  markdown: PropTypes.shape({
-    headers: PropTypes.object.isRequired,
-  }).isRequired,
-};
+export { default } from 'docs/src/modules/components/ComponentLinkHeader';
```

Comment:
Why are this and `ComponentPageTabs` not exported from `@mui/doc`?

Reply:
They must be present locally as they are included in markdown files (with the `{"component": ...` syntax)